settings.gradle.kts

```diff
@@ -255,8 +255,8 @@ include(":instrumentation:mongo:mongo-async-3.3:javaagent")
 include(":instrumentation:mongo:mongo-common:testing")
 include(":instrumentation:netty:netty-3.8:javaagent")
 include(":instrumentation:netty:netty-4.0:javaagent")
-include(":instrumentation:netty:netty-4.1:library")
 include(":instrumentation:netty:netty-4.1:javaagent")
+include(":instrumentation:netty:netty-4.1-common:javaagent")
```

Comment:
can `netty-4.1-common:javaagent` be merged into `netty-4.1:javaagent` and have reactor-netty:javaavent depend directly on `netty-4.1:javaagent`? or is there advantage to keeping it?

Reply:
Yes, it can! I don't know why I haven't thought of that 🙈 
Done.

Reply:
I had to go back to the `netty-4.1-common` idea, muzzle failed when I merged that into the main 4.1 module: 
```
MUZZLE PASSED io.opentelemetry.javaagent.instrumentation.netty.v4_1.NettyInstrumentationModule BUT FAILURE WAS EXPECTED
```